### DISCUSSION TITLE
Why task name in the copied workflow add copy suffix

### DISCUSSION THREAD

**Discussion by zhuxt2015** [apache/dolphinscheduler#10610] [Q&A]
I found that in the import and copy workflow, the task name is appended with the suffix of _copy_{current timestamp}, I wonder why it is designed this way, can remove the copy suffix? 
in most cases, the task name does not have a conflict problem in the workflow, the user wants to reuse the original task name

-------------
我发现在导入和复制的工作流中, 任务名字加上了_copy_{当前时间戳}的后缀, 我想知道为什么这么设计, 可不可以去掉copy后缀, 大部分情况下,任务名字在工作流中不存在冲突的问题的,用户想复用原来任务名字.

**Comment by zhongjiajie** [comment]
related PR https://github.com/apache/dolphinscheduler/pull/10607, and I already add my own view here

**Comment by DoctorSteveZhou** [comment]
有没有好的方式可以处理该问题，我现在也需要手动删除后缀，太麻烦了
